packages/dds/sequence/src/test/partialLoad.spec.ts

```diff
@@ -144,187 +144,4 @@ describe("SharedString Partial Load", () => {
 
 		assert.equal(localSharedString.getText(), remoteSharedString.getText());
 	});
-
```

Comment:
could you update the `describe` block name and the file name? Perhaps `summaryLoad.spec.ts` or something to that effect.